packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -207,9 +207,13 @@ class WebViewController {
 
   /// Accessor to the current URL that the WebView is displaying.
   ///
-  /// If initialUrl was never specified, currentUrl returns `null`.
+  /// If [WebView.initialUrl] was never specified, or if , currentUrl returns `null`.
```

Comment:
uber nit: extra space before comma (`...or if , currentUrl...`)

I'm not sure what "if currentUrl returns `null`" means... you mean if the method channel call returns null ?
I feel like we should try to make the documentation here be all the user needs to read. I'd document any relevant platform-specific behavior here, and treat the method channel API as a private implementation detail as much as possible.

Reply:
oops that was a text-editing fail on my part. I meant "If [WebView.initialUrl] was never specified, returns `null`." Fixed!